mvlearn/utils/plotting.py

```diff
@@ -0,0 +1,49 @@
+import matplotlib.pyplot as plt
+import seaborn as sns
+from .utils import check_Xs_y
+from .utils import check_Xs
+
+
+def plot_2views(
+    Xs,
+    y=None,
+    figsize=(10, 10),
+    title="",
+    show=True,
+    context="notebook",
+    equal_axes=False,
+    scatter_kwargs={},
+    fig_kwargs={}
+):
+    r"""
+
+    """
+    if y is None:
+        Xs = check_Xs(Xs)
+    else:
+        Xs,y = check_Xs_y(Xs, y)
+
+    n = Xs[0].shape[1]
+    fig, axes = plt.subplots(n, n, figsize=figsize, **fig_kwargs)
```

Comment:
While I think the default should this, which is a grid of figures for every dimension in X, I think it would be great if there were an option for the user to input a list of indices that they want to plot (so like if they have high dimensional data but they know only that features 0, 1, and 5 are potentially related, they could input an optional list [0, 1, 5] and then this would only make a 3x3 grid plotting those features against each other. Cuz if people have high dimensional data this plotting function won't be as useful

Reply:
Added, good point